opsdroid/core.py

```diff
@@ -494,18 +494,18 @@ def _connector_names(self):  # noqa: D401
 
             tasks.append(self.eventloop.create_task(parse_always(self, event)))
 
-            unconstrained_skills = await self._constrain_skills(self.skills, event)
-            ranked_skills = await self.get_ranked_skills(unconstrained_skills, event)
-            if ranked_skills:
-                tasks.append(
-                    self.eventloop.create_task(
-                        self.run_skill(
-                            ranked_skills[0]["skill"],
-                            ranked_skills[0]["config"],
-                            ranked_skills[0]["message"],
-                        )
+        unconstrained_skills = await self._constrain_skills(self.skills, event)
+        ranked_skills = await self.get_ranked_skills(unconstrained_skills, event)
+        if ranked_skills:
+            tasks.append(
+                self.eventloop.create_task(
+                    self.run_skill(
+                        ranked_skills[0]["skill"],
+                        ranked_skills[0]["config"],
+                        ranked_skills[0]["message"],
```

Comment:
Why has this indentation been removed here? Ranking should only happen on `Message` events.

Reply:
Without this change, the `match_event` matcher is not working. Not sure if I'm missing something else.

Reply:
Hmm it looks like `match_event` is being included in the NLU ranking, which it shouldn't. However this change breaks all the NLU ranking. I'll look at raising a PR to fix this.

Reply:
Will be fixed in #1286

Reply:
Ok that PR is in now so could you update this with the master branch and restore this indentation.